modules/httpcache/httpcache.go

```diff
@@ -26,7 +26,7 @@ func GetCacheControl() string {
 // generateETag generates an ETag based on size, filename and file modification time
 func generateETag(fi os.FileInfo) string {
 	etag := fmt.Sprint(fi.Size()) + fi.Name() + fi.ModTime().UTC().Format(http.TimeFormat)
-	return base64.StdEncoding.EncodeToString([]byte(etag))
+	return `"` + base64.StdEncoding.EncodeToString([]byte(etag)) + `"`
```

Comment:
This seems like a worthwile change as [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag) seems strict about these quotes. I was under the impression that it did work before so maybe some browsers do not strictly require them.